dotrix_core/src/renderer/backend.rs

```diff
@@ -89,6 +89,14 @@ impl Context {
         }
     }
 
+    pub(crate) fn drop_pipeline(&mut self, shader: Id<Shader>) {
+        self.pipelines.remove(&shader);
+    }
+
+    pub(crate) fn drop_all_pipelines(&mut self) {
+        self.pipelines.clear();
```

Comment:
I think that setting of `self.loaded = false` here and in `drop_pipeline` has sense. What do you think?

Reply:
In `drop_pipeline` I think not because I may be dropping a shader I have deleted from assets, all the others are loaded fine. But that cannot be certain. Perhaps the safe thing would be to force a reload too

Reply:
I'll add it in as it is safer